src/App.js

```diff
@@ -1,13 +1,15 @@
 import './App.css';
 import { Route, Routes } from 'react-router-dom';
 import Homepage from './pages/homepage';
+import QuizPage from './pages/quiz-page';
```

Comment:
nothing wrong with this but as your app grows you would have multiple pages and importing them becomes tedious, therefore consider making an index.js file in your pages folder where you would export all pages from there and import all pages where ever you need them in a single import line.

Reply:
Hey this makes a lot of sense, I'll connect with you on discord to understand this better